spec/openapi.yaml

```diff
@@ -458,6 +514,40 @@ components:
       in: header
       name: x-api-key
   schemas:
+    AddSetTagPropertiesRequest:
+      type: object
+      properties:
+        tagName:
+          type: string
+          description: The name of the tag that will be contain the tag property.
+        tagProperties:
+          type: object
+          description: A set of tag properties.
+          additionalProperties:
+            type: string
+          example:
+            PRIMARY_GROUP: billable/non-billable
```

Comment:
we should use lowercase in example, and mention in MetadataSet (see other comment) that all tag property names are converted/stored in lowercase.